ui/app/components/calendar-widget.js

```diff
@@ -0,0 +1,176 @@
+import Component from '@glimmer/component';
+import layout from '../templates/components/calendar-widget';
+import { setComponentTemplate } from '@ember/component';
+import { format, formatRFC3339 } from 'date-fns';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+// ARG TODO documentation takes start and end for handQuery
+/**
+ * @module CalendarWidget
+ * CalendarWidget components are used in the client counts metrics. It helps users understand the ranges they can select.
+ *
+ * @example
+ * ```js
+ * <CalendarWidget
+ * @param {function} handleQuery - calls the parent pricing-metrics-dates handleQueryFromCalendar method which sends the data for the network request.
+ * @param {object} startDate - The start date is calculated from the parent. This component is only responsible for modifying the end Date. ANd effecting single month.
+ * />
+ *
+ * ```
+ */
+
+const MONTH_MAPPING = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+];
+
+class CalendarWidget extends Component {
+  currentDate = new Date();
+  currentYear = this.currentDate.getFullYear(); // integer
+  currentMonth = parseInt(format(this.currentDate, 'M')) - 1; // integer and zero index
```

Comment:
It may be cleaner to use `this.currentDate.getMonth()` here? Since all we want is the integer

Reply:
Good call, it also seems to return as zero index. Didn't know that was a Date FNS method, now I do :)